src/misc/SvgPolyfill.ts

```diff
@@ -0,0 +1,11 @@
+import { InitializationEvents } from '../events/InitializationEvents';
+import { $$ } from '../utils/Dom';
+require('svg4everybody');
```

Comment:
Actually, we will not need this polyfill anymore since we will not use the `use` tag. Icon with filetypes will exclusively be served via css.